pkgs/applications/networking/instant-messengers/signalbackup-tools/default.nix

```diff
@@ -1,6 +1,6 @@
-{ lib, stdenv, clang14Stdenv, fetchFromGitHub, openssl, sqlite }:
+{ lib, stdenv, llvmPackages_14, fetchFromGitHub, openssl, sqlite }:
 
-(if stdenv.isDarwin then clang14Stdenv else stdenv).mkDerivation rec {
+(if stdenv.isDarwin then llvmPackages_14.stdenv else stdenv).mkDerivation rec {
```

Comment:
See #219431 for why this is important.

Reply:
The naming is rather unfortunated here because on linux clang14Stdenv is an alias.